_tutorials/convert-html-site-to-jekyll.md

```diff
@@ -0,0 +1,481 @@
+
+## 6. Configure site variables
+
+You already configured the page title using `{% raw %}{{ page.title }}{% endraw %}` tags. But there are more `title` tags to populate. Pages also have a [`title`](https://moz.com/learn/seo/title-tag) tag that appears in the browser tab or window. Typically you put the page title followed by the site title here.
+
+In your `default.html` layout, look for the `title` tags below your `head` tags:
+
+```
+<title></title>
+```
+
+Insert the following site variables:
+
+```
+{% raw %}<title>{{ page.title }} | {{ site.name }}</title>{% endraw %}
+```
+
+Open `_config.yml` and add a `name` property for your site's name.
+
+```
+name: My Awesome site
+```
+
+Any properties you add in your `_config.yml` file are accessible through the `site` namespace. Similarly, any properties in your page's front matter are accessible through the `page` namespace. Use dot notation after `site` or `page` to access the value.
+
+Stop your Jekyll server (**Ctrl + C**) and restart it. Verify that the `title` tags are populating correctly. (Every time you modify your config file, you have to restart Jekyll for the changes to take effect. When you modify other files, Jekyll automatically picks up the changes when it rebuilds.)
+
+If you have other variables to populate in your site, do so following this same pattern.
+
+## 7. Show posts on a page
+
+It's common to show a list of posts on the homepage. First, let's create some posts so that our loop will have something to display.
+
+Add some posts in a `_posts` folder following the standard `YYYY-MM-DD-title.md` post format:
+
+* `2017-01-02-my-first-post.md`
+* `2017-01-15-my-second-post.md`
+* `2017-02-08-my-third-post.md`
+
+In each post, add some basic content:
+
+```
+---
+title: My First Post
+layout: default
+---
+
+Some sample content...
+```
+
+Now let's create a layout that will display the posts. Create a new file in `_layouts` called `home.html`. In your `home.html` layout, add the following logic:
+
+```
+---
+layout: default
+---
+
+{% raw %}<ul class="myposts">
+{% for post in site.posts %}
+    <li><a href="{{ post.url }}">{{ post.title}}</a>
+    <span class="postDate">{{ post.date | date: "%b %-d, %Y" }}</span>
+    </li>
+{% endfor %}
+</ul>{% endraw %}
+```
+
+Create a file called `blog.md` in your root directory and specify the `home` layout:
+
+```
+---
+title: Blog
+layout: home
+---
+```
+
+When a layout specifies another layout, it means the content of the first layout will be stuffed into the `{% raw %}{{ content }}{% endraw %}` tag of the second layout. As an analogy, think of Russian dolls that fit into each other. Each layout fits into another layout that it specifies.
+
+```
+home --> default
```

Comment:
updated. I'll fix this confusion with a diagram instead. My arrow was a lame attempt at an illustration.